scripts/pdf/render.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+#A Python script to generate a single PDF document with all the tldr pages. It works by generating 
+#intermediate HTML files from existing md files using Python-markdown, applying desired formating 
+#through CSS, and finally rendering them as PDF. There is no LaTeX dependency for generating the PDF.
+
+import os
+import sys
+import glob
+import re
+import markdown
+import argparse
+
+from weasyprint import HTML
+
+def main(loc, colorscheme):
+
+    oslist = []
+    allmd = []
+    group = []
+    ap = []
+
+    #Checking correctness of path
+    if not os.path.isdir(loc):
+        print("Invalid directory. Please try again!", file = sys.stderr)
+        sys.exit(1)
+
+    if not loc.endswith('/'):
+        loc = loc + "/"
+
+    #Writing names of all directories inside 'pages' to a list
+    for os_dir in os.listdir(loc):
+        oslist.append(os_dir)
+
+    oslist.sort()
+
+    #Required strings to create intermediate HTML files
+    header = "<html><head><link rel=\"stylesheet\" type=\"text/css\" href=\"" + colorscheme + ".css\"></head><body>\n"
+    footer = "</body></html>"
+    title_content = "<h1 class=\"titlemain\">tldr pages</h1><h4 class=\"titlesub\">Simplified and community driven man pages</h4></body></html>"
+
+    #Creating title page
+    with open("title.html", 'w') as f:
+        f.write(header + title_content)
+
+    group.append(HTML('title.html').render())
+
+    for operating_sys in oslist:
+
+        i = 1
+
+        #Required string to create directory title pages
+        dir_title = "<h2 class=\"titledir\">" + operating_sys.capitalize() + "</h2></body></html>"
```

Comment:
Follow the same convention as before - either use triple quotes, or remove the double-quotes.

Reply:
Removed the quotes